### ISSUE TITLE
diff_loss error while using GPU

### ISSUE THREAD

**Issue by bukhalmae145** [PlayVoice/so-vits-svc-5.0#140]
When I train the model, from ./grad_extend/train.py line 135 uses loss.backward(). Other losses(prior_loss, mel_loss, etc..) works fine with loss.backward(), but only diff_loss makes error with using GPU(MPS) on GradTTS. (CPU works fine with it)
`/Users/workstation/Music/Grad-SVC V3 CFM/grad/encoder.py:181: UserWarning: MPS: The constant padding of more than 3 dimensions is not currently supported natively. It uses View Ops default implementation to run. This may have performance implications. (Triggered internally at /Users/runner/work/pytorch/pytorch/pytorch/aten/src/ATen/native/mps/operations/Pad.mm:474.)
  x = torch.nn.functional.pad(x, convert_pad_shape([[0,0],[0,0],[0,0],[0,1]]))
/AppleInternal/Library/BuildRoots/afeb0064-69b1-11ee-9dd8-926038f30c31/Library/Caches/com.apple.xbs/Sources/MetalPerformanceShaders/MPSNDArray/Kernels/MPSNDArrayConvolutionA14.mm:4352: failed assertion `destination kernel width and filter kernel width mismatch'
[1]    15042 abort      python gvc_trainer.py`
